enhancements/oc/must-gather-operator-pvc-destination.md

```diff
@@ -0,0 +1,262 @@
+---
+title: must-gather-operator-ftphost
+authors:
+  - "@shivprakashmuley"
+reviewers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+api-approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+creation-date: 2025-09-04
+last-updated: 2025-09-04
+tracking-link:
+  - https://issues.redhat.com/browse/MG-68
+
+---
+
+# Must-Gather Operator: PVC destination for gathered data
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in `openshift-docs`
+
+## Summary
+
+Introduce a required field in the `MustGather` custom resource that specifies a PersistentVolumeClaim (PVC) as the destination for gathered artifacts. The must-gather operator mounts the referenced PVC into the gather pod so that all content written to `/must-gather` is persisted to the PVC.
```

Comment:
should this be required or optional? can we just default to ephemeral and enable pvcs only if necessary?